build.gradle.kts

```diff
@@ -183,3 +183,20 @@ subprojects {
         }
     }
 }
+
+/*
+ * Javadoc
+ * ====================================================
+ *
+ * Build an aggregate javadoc for all subprojects.
+ */
+if (plugins.hasPlugin("java")) {
```

Comment:
Since this is the root project, we don't need this if statement like we do for subprojects